apps/web/app/(public)/(projects)/projects/projects-page.tsx

```
@@ -65,11 +73,110 @@ export default function ProjectsPage() {
+              size="sm"
+              onClick={() => setPage(String(pageNumber + 1))}
+              disabled={!data.pagination.hasNextPage}
+              className="rounded-none border border-neutral-800 bg-neutral-900 px-3 py-2 text-sm text-neutral-300 hover:border-neutral-700 disabled:opacity-50"
+            >
+              Next
+              <ChevronRight className="h-4 w-4" />
+            </Button>
+          </div>
+        )}
+
+        {data && data.pagination && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)